src/components/List/SimpleList/SimpleList.test.jsx

```diff
@@ -67,16 +72,28 @@ describe('SimpleList component tests', () => {
     expect(getByTitle('Item 5')).toBeTruthy();
   });
 
-  test('SimpleList handleSelect', () => {
+  test('the first item is selectable via keyboard', () => {
     const { getAllByRole } = render(<SimpleList title="Simple list" items={getListItems(1)} />);
     fireEvent.keyPress(getAllByRole('button')[0], { key: 'Enter', charCode: 13 });
+    expect(getAllByRole('button')[0]).toBeInTheDocument();
+    expect(getAllByRole('button')[0]).toBeVisible();
+    expect(
+      getAllByRole('button')[0].className.includes(`${iotPrefix}--list-item__selected`)
+    ).toEqual(true);
   });
 
-  test('SimpleList when unselect selected item', () => {
-    const { getAllByRole } = render(<SimpleList title="Simple list" items={getListItems(1)} />);
-    fireEvent.keyPress(getAllByRole('button')[0], { key: 'Enter', charCode: 13 });
-    fireEvent.keyPress(getAllByRole('button')[0], { key: 'Enter', charCode: 13 });
-  });
+  // this is a failing test, opened defect at https://github.com/IBM/carbon-addons-iot-react/issues/1149
```

Comment:
I think so! 😬  I'm surprised we haven't had a defect opened on us about it.

As you can see, I refactored the test so that it fails - left it there but just commented it out for once we fix it.

It's a good example of why all tests should have at least 1 assertion, which we will now enforce via the new global `beforeEach()` in `setupTest.js` in this PR. Just rendering and firing events isn't really enough.

Reply:
Great catch. This has definitely changed how I look at tests now

Reply:
Same. I'll be looking at tests in PR's with whole new eyes.